clang/include/clang/Interpreter/CodeCompletion.h

```diff
@@ -23,8 +23,13 @@ namespace clang {
 class CodeCompletionResult;
 class CompilerInstance;
 
-void codeComplete(CompilerInstance *InterpCI, llvm::StringRef Content,
-                  unsigned Line, unsigned Col, const CompilerInstance *ParentCI,
-                  std::vector<std::string> &CCResults);
+struct ReplCodeCompletion {
```

Comment:
Can we rename that as `CodeCompleter` or similar and add some doxygen-style documentation?